woocommerce-services.php

```diff
@@ -788,8 +788,16 @@ private function dismiss_tos_notice() {
 		 * @return bool
 		 */
 		private function can_accept_tos() {
+			if ( defined( 'JETPACK_DEV_DEBUG' ) && JETPACK_DEV_DEBUG && current_user_can( 'manage_options' ) ) {
+				return true;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
```

Comment:
@DanReyLop The issue is that in `check_jetpack_install()` we return true if the php constant is defined, regardless of whether or not jetpack is installed.

Actually, maybe it's better to fix `check_jetpack_install()`... I'll take a shot at that.